<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonClickMe">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>560</y>
     <width>95</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>347</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="Image_True">
      <property name="text">
       <string>Richtig</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Image_Falsche">
      <property name="text">
       <string>Falsch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Image_Noise">
      <property name="text">
       <string>Rauschen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="choosenFile">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ausgew√§hlte Datei</string>
   </property>
  </widget>
  <widget class="QLabel" name="Image_Label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Vorhersage">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
